boilerplate/package.json

```diff
@@ -131,29 +132,31 @@
   },
   "detox": {
     "test-runner": "jest",
+    "runnerConfig": "./detox/config.json",
+    "specs": "detox",
     "configurations": {
       "ios.sim.debug": {
         "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/HelloWorld.app",
-        "build": "xcodebuild -workspace ios/HelloWorld.xcworkspace -scheme HelloWorld -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build -destination 'name=iPhone 11'",
+        "build": "xcodebuild -workspace ios/HelloWorld.xcworkspace -scheme HelloWorld -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build -destination 'name=iPhone 13'",
         "type": "ios.simulator",
         "device": {
-          "name": "iPhone 11",
-          "os": "iOS 13.2"
+          "name": "iPhone 13",
+          "os": "iOS 15.5"
         }
       },
       "ios.sim.release": {
         "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/HelloWorld.app",
-        "build": "xcodebuild -workspace ios/HelloWorld.xcworkspace -scheme HelloWorld -configuration Release -sdk iphonesimulator -derivedDataPath ios/build -destination 'name=iPhone 11'",
+        "build": "xcodebuild -workspace ios/HelloWorld.xcworkspace -scheme HelloWorld -configuration Release -sdk iphonesimulator -derivedDataPath ios/build -destination 'name=iPhone 13'",
         "type": "ios.simulator",
         "device": {
-          "name": "iPhone 11",
-          "os": "iOS 13.2"
+          "name": "iPhone 13",
+          "os": "iOS 15.5"
```

Comment:
I would prefer that we not have to hardcode these, but we'll have to look at this later.